### PR TITLE
[v15] Use download center route for cloud clusters download link in support page

### PR DESCRIPTION
**Pull request by mcbattirola**
Backport #36804 to branch/v15

### CODE REVIEW

**Review by avatus — APPROVED** [review]

File: web/packages/teleport/src/Support/__snapshots__/Support.story.test.tsx.snap
Comment (kimlisa) [inline web/packages/teleport/src/Support/__snapshots__/Support.story.test.tsx.snap:272]: omg i just noticed this. this is routing in-app right? we need to use the react-router-dom `Link` so that navigating to this doesn't re-load the entire app. Also we probably dont want to open this in another tab right?

Comment (avatus) [reply web/packages/teleport/src/Support/__snapshots__/Support.story.test.tsx.snap:272]: `Link` compiles down to `a` tags, and this is the snapshot. I don't think this will open in another tab because of the relative path (which loads like the `Link` correctly). you can test it by opening the web app, adding the `target="_blank"` attribute to any of our links and it works fine

Comment (mcbattirola) [reply web/packages/teleport/src/Support/__snapshots__/Support.story.test.tsx.snap:272]: It looks like this is really opening in a new tab, at least this is how its behaving in my browser.

I'll open a PR on master to use a `react-router-dom` Link here, then it should work as expected.

I'll wait to merge this backport once the fix is merged, so I backport both together

**Review by kimlisa — APPROVED** [review]

**Pull request merged**
